lib/src/main/java/growthbook/sdk/java/GbConstants.java

```diff
@@ -0,0 +1,6 @@
+package growthbook.sdk.java;
+
+interface GbConstants {
+    String ENABLED = "enabled";
+    Integer QUANTITY_TO_CUT_SSE = 5;
+}
```

Comment:
Can we move the `QUANTITY_TO_CUT_SSE` to repository and ENABLED to transformer util class?